analyzer/codechecker_analyzer/analyzers/clangtidy/config_handler.py

```diff
@@ -27,7 +27,12 @@ def set_checker_enabled(self, checker_name, enabled=True):
         """
         Enable checker, keep description if already set.
         """
-        if checker_name.startswith("Wno-") or checker_name.startswith("W"):
+        if checker_name == 'clang-diagnostic-error':
+            self.add_checker(checker_name)
+        elif checker_name.startswith('clang-diagnostic-'):
```

Comment:
Please update the user guide and the command line help that from now we can enable/disable warnings on this way.
https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/user_guide.md#toggling-compiler-warnings-

Also I tried to run the following command:
```
CodeChecker analyze -o ./reports ./build.json -c -d clang-diagnostic-unused-variable
```
on a simple source file:
```
int main() { int x = 42; }
```
And I get the following warning messages:
```
[WARNING 2020-04-07 09:01] - No checker(s) with these names was found:
clang-diagnostic-unused-variable
[WARNING 2020-04-07 09:01] - Please review the checker names.
In the next release the analysis will not start with invalid checker names.
```
If I enable/disable the warnings by using the following format: `-d Wunused-variable` or `-e Wunused-variable` I do not see these warnings.